apps/ui/src/composables/useSpaceAlerts.ts

```diff
@@ -4,8 +4,11 @@ import {
   OVERRIDING_STRATEGIES
 } from '@/helpers/constants';
 import { offchainNetworks } from '@/networks';
+import { useRelayerInfoQuery } from '@/queries/relayerInfo';
 import { Space } from '@/types';
 
+const ETH_SIGNATURE_AUTHENTICATOR =
+  '0x95CF9B585fDb12DeB78002B5643dFF8fe67a496D';
```

Comment:
We shouldn't hardcode the address here, can't we take it from constants or sx.js? This also assume all sig authenticator addresses on all network will be the same in future, which is not ideal. And it doesn't support Starknet